geocity/settings.py

```diff
@@ -26,11 +26,25 @@
 CLEAR_PUBLIC_SCHEMA_ON_FIXTURIZE = os.getenv("CLEAR_PUBLIC_SCHEMA_ON_FIXTURIZE")
 
 
-SECURE_PROXY_SSL_HEADER = (
-    tuple(os.getenv("SECURE_PROXY_SSL_HEADER").split(","))
-    if os.getenv("SECURE_PROXY_SSL_HEADER")
-    else None
-)
+ENABLE_SSL = os.getenv('ENABLE_SSL', False) == 'True'
+if ENABLE_SSL:
+    SECURE_PROXY_SSL_HEADER = (
+        tuple(os.getenv("SECURE_PROXY_SSL_HEADER").split(","))
+        if os.getenv("SECURE_PROXY_SSL_HEADER")
+        else None
+    )
+    #SECURE_PROXY_SSL_HEADER = ('HTTP_X_FORWARDED_PROTO', 'https')
+    SECURE_SSL_REDIRECT = True
+    SESSION_COOKIE_SECURE = True
+    CSRF_COOKIE_SECURE = True
+
+# https://stackoverflow.com/a/71194288/6630397
+if ENABLE_SSL:
+    CSRF_TRUSTED_ORIGINS = [
+        f"https://*.{os.getenv('BASE_URL')}",
+        "https://*.127.0.0.1",
+    ]
+
```

Comment:
SSL MUST be enabled by default, thus, invert logic

Reply:
Fixed by commits:
- https://github.com/nicolas-heigvd/geocity/commit/e52b53831ac54f20aef2094ea4e804b461d2a238
- https://github.com/nicolas-heigvd/geocity/commit/40698e3b0d28ad7707c4e00ffd9259b87bad7158